.travis.yml

```diff
@@ -57,5 +57,10 @@ script:
 
 - >
   echo;
-  echo "Running PHP_CodeSniffer";
-  composer ci:php:sniff;
+  if php -v | grep -Pq 'PHP\s++(?:[789]|\d{2})'; then
+    echo "Running PHP_CodeSniffer";
+    composer ci:php:sniff;
+  else
+    echo "Skipped PHP_CodeSniffer due to insufficient PHP version";
+    php -v | grep -Po 'PHP\s++[56]\.(\d++\.)*+\d++(?:-\w++)?+';
```

Comment:
I propose moving this to a composer script, e.g., `php:version`. This will make the purpose of this line more obvious to the reader.

Reply:
Have done so.  It was not also possible to add a version comparison function as a Composer script, as Composer just tacks parameters on the end of the command, so this is in the travis.yml script.